### ISSUE TITLE
Wildcard support

### ISSUE THREAD

**Issue by XFox111** [wdzeng/firefox-addon#7]
## Description
It would be nice if firefox-addon could find extension files with a wildcard pattern. For example:

```yml
      - uses: wdzeng/firefox-addon@v1.0.5
        with:
          addon-guid: ${{ secrets.FIREFOX_EXT_UUID }}
          xpi-path: my-extension-*-firefox.zip
          jwt-issuer: ${{ secrets.FIREFOX_API_KEY }}
          jwt-secret: ${{ secrets.FIREFOX_CLIENT_SECRET }}
```

## Proposed solution
The `xpi-path` variable should accept wildcard patterns and search for a file that meets the pattern criteria. If multiple files are found, it should issue an error, since there can only be one file that gets to be uploaded.

## Justification
I've recently migrated my extension to a new framework that has a built-in packaging solution. The problem is: it appends extension's version to the file name (e.g. `my-extension-4.0.0-firefox.zip`). It would be a lot easier if I could use wildcard pattern, rather than updating the workflow and trying to retrieve extension's version there to complete the name.

I assume that this also applies to `edge-addon` and `chrome-extension` actions, but I won't create issues for them yet to avoid duplication

I can work on a PR for this issue if that's ok

**Comment by wdzeng** [comment]
#8 is merged into the main branch.

**Comment by wdzeng** [comment]
Hi @XFox111, I've published this feature in the action v1.1. Thanks for your help!
